docs/index.rst

```diff
@@ -52,6 +52,7 @@ Contents
    install.rst
    getstart.rst
    tutorial.rst
+   tutorial2.rst
```

Comment:
is this really the correct name?  I don't see an actual `tutorial2.rst` file anyplace